docs/quick-start/README.md

```diff
@@ -289,6 +289,22 @@ Error: Configuration property "app.name" is not defined
 
 You can resolve this by running `npx mira init` or checking your configuration file matches the expected fields. 
 
+### CDK Deployment Issues
+
+Deployment of Mira applications requires a bootstrapped CDK environment. This is created when you run the bootstrap command during initial set up (`cdk bootstrap`). This command creates a workspace where temporary assets and configurations can be stored during deployment. On rare occasions this work space can become corrupted and cause strange or unexplained failures such as permission issues. For example, on the S3 Sample application, it can manifest as a permission issue when attempting to copy files to the destination S3 bucket even though the deployment user may have administrator privilege.
```

Comment:
```suggestion
Deployment of Mira applications requires a bootstrapped CDK environment. This is created when you run the bootstrap command during initial set up (`cdk bootstrap`). This command creates a workspace where temporary assets and configurations can be stored during deployment. On rare occasions this workspace can become corrupted and cause strange or unexplained failures such as permission issues. For example, on the S3 Sample application, it can manifest as a permission issue when attempting to copy files to the destination S3 bucket even though the deployment user may have administrator privilege.
```

Reply:
just removed a space